packages/integrations/prefetch/src/client.ts

```diff
@@ -44,12 +44,11 @@ function onLinkEvent({ target }: Event) {
 
 async function preloadHref(link: HTMLAnchorElement) {
 	unobserve(link);
-
 	const { href } = link;
 
 	try {
 		const contents = await fetch(href).then((res) => res.text());
-		parser = parser || new DOMParser();
+		parser ||= new DOMParser();
```

Comment:
checked and this seems safe to use: https://caniuse.com/mdn-javascript_operators_logical_or_assignment